tools/clean

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# A basic script to clean all of the build artifacts and installed
+# modules.  Order matters, and it's easy to overlook something. In the
+# future this could be structured to be done via a single
+# one-build-tool-to-rule-them-all command, but currently we've a
+# plethora of tooling and this is at least a starting point.
+
+MAG="\033[95m"
+END="\033[0m"
+
+status() {
```

Comment:
Let's also include a `fail` command. Below, on each clean command, we can provide verbose failures.

Reply:
Done.